package.json

```diff
@@ -10,14 +10,19 @@
     "eslint-plugin"
   ],
   "author": "Chris Sauve <chris.sauve@shopify.com>",
+  "engines": {
+    "node": ">=6.11.1"
+  },
   "scripts": {
     "check": "yarn run lint && yarn test",
     "rules-status": "eslint-index lib/config/all.js  --format table",
     "rules-omitted": "eslint-index lib/config/all.js  --status omitted",
     "lint": "eslint . --max-warnings 0",
     "test": "NODE_PATH=$NODE_PATH:./transforms:./tests mocha 'tests/**/*.js' --reporter spec --compilers js:babel-core/register",
     "test:watch": "yarn test -- --watch --reporter min",
-    "test:cover": "babel-node node_modules/.bin/isparta cover --report text --report html _mocha -- --reporter spec --recursive tests/"
+    "test:cover": "babel-node node_modules/.bin/isparta cover --report text --report html _mocha -- --reporter spec --recursive tests/",
+    "preversion": "yarn run check",
+    "postversion": "git push && git push --tags"
```

Comment:
Ahh okay, my concern is that code/tags accidentally gets pushed when a dev does some version tweaking locally. 

Eg. Accidentally doing a minor bump instead of a patch.  


Also, I don't think the preversion is necessary. We deploy through shipit, which expects CI to pass before deploying. Our CI already runs `yarn run check`.